.github/workflows/build.yml

```diff
@@ -95,6 +97,15 @@ jobs:
           cache: 'yarn'
       - name: Install dependencies with yarn
         run: yarn install --frozen-lockfile
+      - name: Print Xcode environment
+        if: startsWith(matrix.os, 'macos-')
+        run: |
+          echo "Xcode Path: $(xcode-select --print-path)"
+          echo "SDK Path: $(xcrun --show-sdk-path)"
+          echo "SDK Version: $(xcrun --show-sdk-version)"
+          echo "SDK Build Version: $(xcrun --show-sdk-build-version)"
+          echo "SDK Platform Path: $(xcrun --show-sdk-platform-path)"
+          echo "SDK Platform Version: $(xcrun --show-sdk-platform-version)"
```

Comment:
l: just checking, is this intentionally kept in the PR or was it for testing CI? No problem if you wanna keep it

Reply:
As these paths could change over time due to update of the CI runner image, I would like to keep them in, so in case our tests are failing, we can directly compare the expected and actual paths.

I'll add this as a comment